examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
+
+## Tutorial
+
+Steps
+1. create IPFS instance
+
+First, we need an IPFS instance that we'll use throughout the tutorial. For the sake of convinience, we'll use `ipfs-daemon` module to create and IPFS instance.
+
+Add `ipfs-daemon` to your project:
+```
+npm i ipfs-daemon --save
+```
+
+Add `ipfs-daemon` to your html:
+```html
+<script type="text/javascript" src="node_modules/ipfs-daemon/dist/ipfs-browser-daemon.js" charset="utf-8"></script>
+```
+
+Create the IPFS instance and wait for it to start:
+
+```javascript
+const ipfs = new IpfsDaemon({
+  // Directory to which save IPFS data to
+  IpfsDataDir: '/path/to/ipfs/data',
+  // IPFS dev server: webrtc-star-signalling.cloud.ipfs.team
+  SignalServer: '188.166.203.82:20000',
+})
+
+ipfs.on('error', (e) => console.error(e))
+ipfs.on('ready', () => {
+  // We're ready to use IPFS!
+})
+```
+
+2. listen for file drop
```

Comment:
Understood.

Somewhere we discussed, and decided, that we do go-ipfs -> js-ipfs-browser -> js-ipfs-browser (stream a video or smth). I believe adding the support for adding files to the browser version is not that much out of scope in terms of work and it adds another important step to the tutorial for one of the most asked questions for someone who's starting with ipfs "how do I add files to IPFS?". So having that functionality gives a more fully fledged tutorial than just catting files from go-ipfs. Don't worry, we're still going for the same goal: access files from go-ipfs.